ui/admin/app/controllers/scopes/scope/workers/index.js

```diff
@@ -16,8 +17,40 @@ export default class ScopesScopeWorkersIndexController extends Controller {
   @service router;
   @service can;
 
+  // =attributes
+
+  @tracked selectedWorker;
+
+  /**
+   * Get the first 10 tags of the selected worker.
+   * @type {object[]}
+   */
+  get selectedWorkerTags() {
+    // only show the first 10 tags if there are more than 10
+    return this.selectedWorker.allTags.slice(0, 10);
+  }
+
+  /**
+   * Build the display name for a worker tag and will
+   * truncate the key if it's too long.
```

Comment:
I think this comment needs to be updated now after the truncation logic was removed

Reply:
Thanks. Updated now.